### ISSUE TITLE
Long long long variable names.

### ISSUE THREAD

**Issue by someq** [MaikuB/flutter_local_notifications#1083]
It's not a bug, but more of enhancement. Variables and classes names are very very very long. 

Consider this config example:

```dart
FlutterLocalNotificationsPlugin flutterLocalNotificationsPlugin =
    FlutterLocalNotificationsPlugin();
const AndroidInitializationSettings initializationSettingsAndroid =
    AndroidInitializationSettings('app_icon');
final IOSInitializationSettings initializationSettingsIOS =
    IOSInitializationSettings(
        onDidReceiveLocalNotification: onDidReceiveLocalNotification);
final InitializationSettings initializationSettings = InitializationSettings(
    android: initializationSettingsAndroid,
    iOS: initializationSettingsIOS);
flutterLocalNotificationsPlugin.initialize(initializationSettings,
    onSelectNotification: onSelectNotification);
```

By replacing `initializationSettings` with just `settings` we get much shorter version. Or we may use `initConfig`, `initSettings`, `initConf`, `config`, etc.

We can remove the word `Flutter` from `FlutterLocalNotificationsPlugin` (obviously it's a flutter plugin) getting a `localNotificationsPlugin` and depending on the code and context shorten it even more to `localPlugin`, `localNotifications` or `notificationPlugin`.

One possible result:

```dart
LocalNotificationsPlugin notificationsPlugin = LocalNotificationsPlugin();
const AndroidInitSettings androidSettings = AndroidInitSettings('app_icon');
final IOSInitSettings iOSSettings = IOSInitSettings(
  onDidReceiveLocalNotification: onDidReceiveLocalNotification,
);
final InitSettings settings = InitSettings(
  android: androidSettings,
  iOS: iOSSettings,
); 
notificationsPlugin.initialize(
  settings,
  onSelectNotification: onSelectNotification,
);
```

Probably it can be wrapped somehow into a more convenient API?

**Comment by MaikuB** [comment]
The measure of convenience you're using seems to be solely based on length. Variable names are up to developers to define and depending on the coding style, will closely match the class names or not even be defined by using an [inline variable](https://refactoring.com/catalog/inlineVariable.html). The classes are defined the way they are for readability in the sense that it's closely matches what would have been expressed in English so it becomes easier to reason with (i.e. initialisation settings related to the act of initialisation by calling `initialize()`) . This isn't something I don't want to sacrifice when many others have used the plugin without complaints on this (note: suggestions made here would be breaking). Furthermore, `init` tends to be associated with an action so it can introduce some confusion/readability issues when `initSettings` is used as a class name. Dropping `Flutter` from the name of the plugin removes the an element of consistency with the name of the plugin. WIthin the ecosystem the plugin class tends to be the same as the name of the package though here I added `Plugin` at the end to so the code wouldn't read like developers were creating a local notification. There is also another plugin called `local_notifications`.

To summarise I don't have plans to make changes on this so will be closing this, though I do wonder what you would think of doing for other packages with class names of similiar length e.g. https://pub.dev/documentation/firebase_messaging_platform_interface/latest/firebase_messaging_platform_interface/AndroidNotificationPriority-class.html

**Comment by someq** [comment]
It's not based on length only but more on readability. IMHO, shorter variant is more readable, while still understandable. But I understand what's seems readable for me may look inconsistent for you and vice versa. Also I understand it's not acceptable to change variable and classes names when lot of people use the plugin already. That's why I propose a wrapper.

What will I do about other packages? Nothing, probably. Complain and write my own wrappers for them.
